tensilelite/Tensile/KernelWriter.py

```diff
@@ -448,8 +448,7 @@ def __init__( self, kernelMinNaming, kernelSerialNaming ):
   ##############################################################################
   def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteEndIter, uDu=0, skipGlobalReadInc=False, firstIter=False, lastLoop=False, lastLc=False):
 
-    currentIsa = globalParameters["CurrentISA"]
-    maxVmcnt = globalParameters["AsmCaps"][currentIsa]["MaxVmcnt"]
+    maxVmcnt = self.ti.getAsmCaps()["MaxVmcnt"]
```

Comment:
here we can git Asmcaps from self.states

Reply:
done